<launch>    
    <!-- Random drive -->
    <node pkg="kjq_navigation" type="random_drive" name="random_drive" output="screen">
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find kjq_navigation)/rviz/test_costmap.rviz" />

</launch>
